libr/arch/p/arch_riscv.c

```diff
@@ -2,13 +2,14 @@
 
 #include <r_lib.h>
 #include <r_asm.h>
-#include <r_anal.h>
+#include <r_arch.h>
 #include "../../asm/arch/riscv/riscv-opc.c"
```

Comment:
those includes should be. moved into libr/arch/p/riscv/*.c

Reply:
Fix here: 2158f3cd11d830eea37c410a4840a942e5d0e086